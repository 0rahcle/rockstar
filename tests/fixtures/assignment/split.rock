Let the string be "a,b,c"
Split the string into pieces
Say pieces at 0
Say pieces at 1
Say pieces at 2

Let the line be "livin' on a prayer"
Split the line into words
Say words at 0
Say words at 1
Say words at 2
Say words at 3
