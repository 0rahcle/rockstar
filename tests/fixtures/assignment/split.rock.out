a
b
c
livin'
on
a
prayer
